src/usr/local/www/interfaces.php

```diff
@@ -632,7 +632,7 @@ function remove_bad_chars($string) {
 			foreach ($ifdescrs as $ifent => $ifdescr) {
 				if ($if != $ifent && ($config[interfaces][$ifent]['ipaddrv6'] == $_POST['type6'])) {
 					if ($config[interfaces][$ifent]['prefix-6rd'] == $_POST['prefix-6rd']) {
-						$input_errors[] = gettext("You can only have one interface configured in 6rd with same prefix.");
+						$input_errors[] = gettext("Only one interface can be configured in 6rd with same prefix.");
```

Comment:
Seems awkward... Perhaps "Only one interface can be configured within a single 6rd prefix" or "An interface already exists using that 6rd prefix" (which implies you cannot have a second)